bootstrap/handlers/httpserver.go

```diff
@@ -139,10 +140,15 @@ func (b *HttpServer) BootstrapHandler(
 
 		b.isRunning = true
 		err := server.ListenAndServe()
-		if err != nil {
+		// "Server closed" error occurs when Shutdown above is called in the Done processing, so it can be ignored
+		if err != nil && err.Error() != "http: Server closed" {
```

Comment:
comparing the error message is fragile
use 
`err != server.ErrServerClosed` instead
https://cs.opensource.google/go/go/+/refs/tags/go1.18.4:src/net/http/server.go;l=2997

Reply:
Great suggestion! THX!

Reply:
Fixed